WinImageBuilder.psm1

```diff
@@ -64,6 +64,10 @@ class PathShouldExistAttribute : System.Management.Automation.ValidateArgumentsA
     }

```

Comment:
For the commit message:
Remove the . from the end of the commit message.

Add to the commit description the steps you are adding logs to.